doc/radosgw/admin.rst

```diff
@@ -100,6 +100,12 @@ For example::
   radosgw-admin subuser create --uid=johndoe --subuser=johndoe:swift --access=full
 
 
+It is preferred to be explicit when creating subusers, specifying users with a ``--uid``.
+If a ``--uid`` is not supplied, we parse the uid until the last colon ::
+
```

Comment:
I think a better phrasing might be

"When creating a subuser, it is best to explicitly indicate the parent user with a ``--uid`` argument.  If ``--uid`` is not supplied, the command expects the ``--subuser`` value to be prefixed with the parent user followed by a colon:

Reply:
@anthonyeleven It's not only about doc! As you can see the diff it's changing the current cli behavior.